dev-tools/ecs-migration.yml

```diff
@@ -90,3 +90,26 @@
   to: source.domain
   alias: true
   copy_to: false
+
+# Metricbeat base fields
+- from: metricset.name
+  to: event.dataset
+  alias: true
+  copy_to: false
+
+- from: metricset.module
+  to: event.module
+  alias: true
+  copy_to: false
+
+# TODO: Is it really an alias? Or will we change the unit?
+- from: metricset.rrt
+  to: event.duration
+  alias: true
+  copy_to: false
+
+# TODO: This becomes port and domain in servce?
```

Comment:
@webmat Feedback welcome on this one.

Reply:
Hmmm, service doesn't have domain, ip or port.

Is the intent to represent the address exposed by a service? E.g.

```YAML
service.name: sshd
service.port: 22
service.ip: 10.10.10.10
service.domain: (hostname on the local network)
```

Reply:
It is the host/hostname that the user configured to connect to the service to fetch the information.

Reply:
Ok, removed from here, got it.

I was thinking of introducing a field for these ambiguous addresses as `address`. It's an extremely generic word, it doesn't clash with any of the words used in the RFC that have a *somewhat* specific meaning like "host", "hostname", or "authority". `address` also kinda works for situations where e.g. NGINX or HAProxy shove a unix socket in the "IP" field.

I think "address" could also work for DSNs like the mysql example above.